contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -60,7 +69,18 @@ def main(argv):
         VerifyTracksExist(track_names=["Page*", "*System*", "*CGroup*"], allow_duplicates=True),
 
         AddIterator(function_name="TestFunc2"),
-        VerifyFunctionCallCount(function_name='TestFunc2', min_calls=1257, max_calls=1257)
+        VerifyFunctionCallCount(function_name="TestFunc2", min_calls=1257, max_calls=1257),
+
+        # Let's take a capture of the current version and verify this can be still loaded
+        EndSession(),
+        ConnectToStadiaInstance(),
+        FilterAndSelectFirstProcess(process_filter="hello_ggp"),
+        Capture(),
+        VerifyTracksExist(track_names="hello_ggp_stand*", allow_duplicates=True),
+        EndSession(),
+        # If we took the capture around midnight, we need to ensure to also look for the next day
+        LoadLatestCapture(filter_strings=[f"hello_ggp_stand_{today_string}", f"hello_ggp_stand_{tomorrow_string}"]),
+        VerifyTracksExist(track_names="hello_ggp_stand*", allow_duplicates=True)
```

Comment:
I think this test is somewhat limited in the signal it gives since you're potentially going back (assuming you actually want to check yesterday, not tomorrow...) up to nearly 48 hours if it's close to midnight. Assuming the tests run nightly at about the same time, you'll always find a capture from the last correct test, so the test will always be false positive for one run. Worst case, it could be flaky every second run and still pass.

As I wrote above, clearing the capture cache beforehand would help. You could also restrict the time to a much smaller window of a few minutes since you know when the capture is supposed to have been saved.

Reply:
As above, it is actually "tomorrow" or today. However, it could still be a catpure from the previous day, but with at most 24 hours going back.

Reply:
Done.